portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -14,19 +14,36 @@
 
 package com.google.sps.servlets;
 
+import com.google.gson.Gson;
 import java.io.IOException;
+import java.util.Arrays;
+import java.util.ArrayList;
+import java.util.List;
 import javax.servlet.annotation.WebServlet;
 import javax.servlet.http.HttpServlet;
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
 
-/** Servlet that returns some example content. TODO: modify this file to handle comments data */
-@WebServlet("/data")
+/** Servlet for handling comments data */
+@WebServlet("/comments")
 public class DataServlet extends HttpServlet {
 
+  private final List<String> testComments = new ArrayList<String>(Arrays.asList( 
+    "This is an example comment.",
+    "Another example comment...",
+    "Wow! Yet another example comment?"));
```

Comment:
I actually thought that you can keep adding the comments to the array list in the init-method, but that is fine as well! So, keep it as is.